modules/miscutil/lib/orcidutils.py

```diff
@@ -671,15 +671,15 @@ def _stub_method():
                                        " doi %s twice." %
                                        (recid, encoded), alert_admin=True)
             else:
-                external_ids.append(('doi', encoded))
+                external_ids.add(('doi', encoded))
```

Comment:
This patch is advertised as fixing the case of duplicate DOIs. However
for the case of duplicate DOI the if clause above at https://github.com/inspirehep/invenio/pull/387/files#diff-5dee2df028678e559448ec75190bd2fdR666  already addresses this case and raises a `DoubleIds` exception which is handled to create an alert. Using a `set` instead of a `list` still makes sense to avoid other duplicate IDs. So do duplicate DOIs actually cause orcidpush issues? If so, why doesn't above work as intended?